### PR TITLE
shellz 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by djerfy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by djerfy** [comment]
> * GitHub repository too new (<30 days old)

It's just new formula file

**Comment by commitay** [comment]
> > * GitHub repository too new (<30 days old)

> It's just new formula file

It's checking this repo: https://github.com/evilsocket/shellz

File: Formula/shellz.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Shellz < Formula
+  desc "Small utility to track and control custom shellz"
+  homepage "https://github.com/evilsocket/shellz"
+  url "https://github.com/evilsocket/shellz/archive/v1.1.0.tar.gz"
+  sha256 "8bec29e04a54adb8b9d161c88422d324e1f3d52d743f4fc0d72ab89ca54f9c1e"
+
+  bottle do
```
Comment (commitay) [inline Formula/shellz.rb]: Remove the bottle block, it will be added when this is merged.

File: Formula/shellz.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    cd "src/github.com/evilsocket/shellz" do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "build"
+      bin.install "shellz"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match "shellz", shell_output("#{bin}/shellz -help 2>&1", 2)
```
Comment (commitay) [inline Formula/shellz.rb]: Please add a better test, e.g. write a minimal config and check that it loads correctly.

**Comment by djerfy** [comment]
I closed this pull request, I review the tests and I would open a new one with the latest version

**Pull request closed without merging**